clang/lib/DPCT/APINamesMemory.inc

```diff
@@ -171,6 +171,41 @@ ASSIGNABLE_FACTORY(CONDITIONAL_FACTORY_ENTRY(
                  CALL(MapNames::getClNamespace() + "malloc_shared",
                       getSizeForMalloc(0, 1), QUEUESTR))))))
 
+CONDITIONAL_FACTORY_ENTRY(
+    UseExtBindlessImages,
+    ASSIGNABLE_FACTORY(DELETER_FACTORY_ENTRY("cudaFreeMipmappedArray", ARG(0))),
+    REMOVE_API_FACTORY_ENTRY_WITH_MSG(
+        "cudaFreeMipmappedArray",
+        "SYCL currently does not support mipmap image type. You "
+        "can migrate the code with bindless images by specifying "
+        "--use-experimental-features=bindless_images."))
```

Comment:
for the string " SYCL currently .......suggest to ....", suggest to define a constant string, then can be used everywhere.

Reply:
fixed